script.js

```diff
@@ -0,0 +1,41 @@
+var tester = function() {
+    console.log('Button Pushed');
+    var fileHeight = document.getElementById('heightNumber').value;
+    console.log(fileHeight);
+    var madeOf = document.getElementById('charType').value;
+    console.log(madeOf);
+        (function () {
+     var spc = [];
+     var leafs = [];
+     var finaltree = []
+
+     for (i = fileHeight; i >=0; i--) {
```

Comment:
Always make sure you initialize your variables with `var` in a for loop: `for (var i = fileheight; ...`